### PR TITLE
Disable additional test case data subjected to IIS local server instance racing

### PR DESCRIPTION
**Pull request by biaol-odata**
For build machine for code coverage debug run:

- SecurityPartialTrustTest
- HttpContextServiceHostTunnelingTest
- RequestUriCaseInsensitive

### Issues

*This pull request fixes issue of test instability due to local IIS server data racing
For build machine for code coverage debug run

- SecurityPartialTrustTest
- HttpContextServiceHostTunnelingTest
- RequestUriCaseInsensitive*

### Description

*All the test cases disabled are confirmed to be related using IIS Local server, along with other server types. Only data related to IIS Local server part is disabled.*

### Checklist (Uncheck if it is not completed)

- [ ] Test cases added
- [ x ] Build and test with one-click build and test script passed

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Tests/SecurityTest.cs [hunk 1]
```diff
@@ -218,15 +218,16 @@ public void SecurityMaxResultsTest()
...
+                            TestUtil.AssertExceptionExpected(exception,
                                 customerCount > maxResultsPerCollection && !linksOnly,
                                 expand && orderCount > maxResultsPerCollection && customerCount > 0);
                         }
                     }
                 });
             }
 
-            [TestMethod]
+            [Ignore] //Removed due to local IIS server instance racing on build machine.
```
Comment (robward-ms) [inline test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Tests/SecurityTest.cs]: Please remove [Ignore] and just leave the comment.

File: test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Tests/RequestUriProcessorTest.cs [hunk 2]
```diff
@@ -370,6 +370,9 @@ public void RequestUriProcessorKeySpecialCharsTest()
                 new Dimension("WebServerLocation", new WebServerLocation[]{
                         WebServerLocation.InProcess,
                         WebServerLocation.InProcessWcf
+
```
Comment (robward-ms) [inline test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Tests/RequestUriProcessorTest.cs]: I don't see anything being removed.

File: test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Tests/SecurityTest.cs (hunk 1 above)
Comment (biaol-odata) [inline test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Tests/SecurityTest.cs]: ok

File: test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Tests/RequestUriProcessorTest.cs (hunk 2 above)
Comment (biaol-odata) [inline test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Tests/RequestUriProcessorTest.cs]: I just added a comment to the file edited @ 72cc649 "remove IIS Local server test cases due to timing issue"

**Pull request merged**